certbot-apache/certbot_apache/_internal/assertions.py

```diff
@@ -36,8 +36,8 @@ def assertEqualComment(first, second): # pragma: no cover
     assert isinstance(first, interfaces.CommentNode)
     assert isinstance(second, interfaces.CommentNode)
 
-    if not isPass(first.comment) and not isPass(second.comment):
-        assert first.comment == second.comment
+    if not isPass(first.comment) and not isPass(second.comment):  # type: ignore
+        assert first.comment == second.comment  # type: ignore
```

Comment:
These were required because mypy is not able to inspect the existence of interface variables and would give an error.